package.json

```diff
@@ -16,18 +16,18 @@
   },
   "dependencies": {
     "color": "^3.0.0",
-    "email-validator": "^1.0.7",
+    "email-validator": "^2.0.3",
     "fp-ts": "^1.4.1",
     "io-ts": "^1.1.1",
     "italia-ts-commons": "^1.7.0",
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.10",
     "native-base": "^2.4.2",
     "native-base-shoutem-theme": "^0.2.2",
     "react": "16.3.0-alpha.1",
     "react-native": "0.54.2",
     "react-native-config": "^0.11.2",
     "react-native-confirmation-code-input": "git://github.com/teamdigitale/react-native-confirmation-code-input.git#550ad82",
-    "react-native-device-info": "^0.13",
+    "react-native-device-info": "^0.21",
```

Comment:
react-native link react-native-device-info?
pod install?

Reply:
sure! I always forget about that step!